geometry_msgs/PoseStamped target_pose
bool do_turning
float32 reaching_radius
